### PR TITLE
Add Fluidd

### PR DESCRIPTION
**Pull request by hasecilu**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/fluidd.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+licenses:
+  - GPL-3.0
+platforms:
+  - TypeScript
+tags:
+  - 3D printing
+source_code_url: https://github.com/fluidd-core/fluidd
+stargazers_count: 1066
+updated_at: '2023-10-03'
+archived: false
```
Comment (nodiscc) [inline software/fluidd.yml:13]:
```suggestion
```

optional field, the default is `false`, let's remove it

File: software/fluidd.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Fluidd
+website_url: https://docs.fluidd.xyz/
+description: Fluidd is a lightweight & responsive user interface for Klipper, the 3D printer firmware.
+licenses:
+  - GPL-3.0
+platforms:
+  - TypeScript
```
Comment (KieranRobson) [inline software/fluidd.yml:7]: > https://docs.fluidd.xyz/installation

```suggestion
platforms:
  - Docker
  - Nodejs
```

File: software/fluidd.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: Fluidd
+website_url: https://docs.fluidd.xyz/
+description: Fluidd is a lightweight & responsive user interface for Klipper, the 3D printer firmware.
```
Comment (KieranRobson) [inline software/fluidd.yml:3]:
```suggestion
description: A ightweight & responsive user interface for Klipper, the 3D printer firmware.
```

File: software/fluidd.yml (hunk 2 above)
Comment (nodiscc) [inline software/fluidd.yml:7]:
```suggestion
  - Docker
  - Nodejs
```

> Values for platform are the main server-side requirements for the software

https://docs.fluidd.xyz/installation

File: software/fluidd.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+licenses:
+  - GPL-3.0
+platforms:
+  - TypeScript
+tags:
+  - 3D printing
+source_code_url: https://github.com/fluidd-core/fluidd
```
Comment (KieranRobson) [inline software/fluidd.yml:10]: > For Klipper

```suggestion
tags:
  - 3D printing
source_code_url: https://github.com/fluidd-core/fluidd
depends_3rdparty: yes
```

**Review by nodiscc — CHANGES_REQUESTED** [review]
Depends on https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/234

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Couple changes recommended.

File: software/fluidd.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+name: Fluidd
+website_url: https://docs.fluidd.xyz/
+description: A ightweight & responsive user interface for Klipper, the 3D printer firmware.
```
Comment (nodiscc) [inline software/fluidd.yml:3]:
```suggestion
description: Lightweight & responsive user interface for Klipper, the 3D printer firmware.
```

typo

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/fluidd.yml (hunk 4 above)
Comment (hasecilu) [reply software/fluidd.yml:11]: Should be Klipper be added in this file?

Comment (nodiscc) [reply software/fluidd.yml:11]: I don't think `depends_3rdparty: yes` applies since Klipper is also Free Software: https://www.klipper3d.org/

**Review by nodiscc — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Comment by nodiscc** [comment]
Rebased, squashed and merged in https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/268. Thanks

**Pull request closed without merging**